src/index.js

```diff
@@ -286,6 +285,7 @@ class WPTS {
 
         const options = seleniumAssistantBrowser.getSeleniumOptions();
         options.addArguments(`user-data-dir=${tempPreferenceFile}/`);
+        options.addArguments(`no-sandbox`);
```

Comment:
No worries. Makes sense to make it optional. I'll try to put together an update in the next few days.